posix/include/sof/lib/mailbox.h

```diff
@@ -72,10 +72,6 @@ void mailbox_dspbox_read(void *dest, size_t dest_size,
 #define mailbox_hostbox_write(_offset, _src, _bytes) \
 	memcpy((char *)ipc->comp_data + _offset, _src, _bytes)
 
-static inline uint64_t mailbox_sw_reg_read64(uint32_t offset)
-{
```

Comment:
Can testbench include this header ? We really want to get rid of platform/include now with Zephyr.

Reply:
The failure is not from testbench source code but rest of SOF code built into testbench.

Reply:
@lgirdwood @singalsu This is not really Zephyr build here, but we have sof/posix implement our RTOS abstraction. I think you could drop "#include <platform/lib/mailbox.h>" from this file to simplify the implementation and have no-op implementations for all mailbox_sw_reg*() functions right here in posix/include/sof/lib/mailbox.h.

But there are many users (one can also built a host/library build of SOF using the Zephyr posix platform), so very easy to miss a case.